sdk/component/internal/storer/local/sqlite/export_test.go

```diff
@@ -0,0 +1,27 @@
+package sqlite
+
+import (
+	"fmt"
+)
+
+// CreateTable is used to create a table in testing settings.
+func (m *manager) CreateTable() error {
+	stmt, err := m.db.Prepare(`
+		CREATE TABLE finding (
+			id INTEGER PRIMARY KEY AUTOINCREMENT UNIQUE,
+			workflow_id UUID NOT NULL UNIQUE ,
+			findings TEXT NOT NULL,
+			created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+			updated_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+		);
+	`)
+	if err != nil {
+		return fmt.Errorf("could not prepare statement for creating table: %w", err)
```

Comment:
Since go 1.13 `errors` and `fmt.Errorf` are sufficient to cover what `go-errors` used to do https://go.dev/play/p/U896OQVWlW4

I'd prefer to not bring in an extra dependency if possible!

Let me know what you think

Reply:
The `github.com/go-errors/errors` is interesting because of 2 reasons:
1. it ensures that the code base is more consistent everywhere because you are not using 2 packages when interacting with errors (the std errors and fmt) but only one which is named errors
2. most important it adds stacktraces to errors which can then be printed from the entrypoint at the top level

Reply:
I can look into this separately after merging these changes if you don't mind https://linear.app/smithy/issue/OCU-268/replace-errors-and-fmterrorf-utilisations-for-go-errors-in-sdk

I don't want to have more changes in the PR :)